Build/PackageFiles/Dotnet_precompiled/ItemTemplates_v3.x.nuspec

```diff
@@ -40,5 +40,6 @@
     <file src="..\..\..\Functions.Templates\Templates\KafkaOutput-CSharp\**" target="content\KafkaOutput-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\SqlInputBinding-CSharp\**" target="content\SqlInputBinding-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\SqlOutputBinding-CSharp\**" target="content\SqlOutputBinding-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\AuthenticationEventsTrigger-CSharp\**" target="content\AuthenticationEventsTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />

```

Comment:
Can you remove the changes to this file. It will light up templates in VS

Reply:
Done